browser/ui-assets.go

```diff
@@ -21,6 +21,9 @@ import (
 	"time"
 )
 
+//lint:file-ignore ST1003 Generated file...
```

Comment:
This is an autogenerated file @klauspost we cannot have this in place it will get overwritten next time ui-assets.go are autogenerated

Reply:
See https://github.com/dominikh/go-tools/issues/429 :( 

It seems like we might be able to exclude it only from the 'browser' directory. Let me try that.

Reply:
Added separate config file instead.